cmd/pdcli/login/cmd.go

```diff
@@ -13,31 +13,168 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
+
 package login
 
 import (
+	"bufio"
+	"errors"
 	"fmt"
+	"os"
 
+	"github.com/PagerDuty/go-pagerduty"
+	"github.com/openshift/pagerduty-short-circuiter/pkg/config"
+	"github.com/openshift/pagerduty-short-circuiter/pkg/pdcli"
 	"github.com/spf13/cobra"
 )
 
-// pdcli/login/cmdCmd represents the pdcli/login/cmd command
+const APIKeyURL = "https://support.pagerduty.com/docs/generating-api-keys#section-generating-a-general-access-rest-api-key"
```

Comment:
Let's take APIKeyURL into a separate package something like `constants` which will hold the constant data. How about `pkg/constants` ?